app/controllers/api/v1/early_career_teacher_participants_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class EarlyCareerTeacherParticipantsController < Api::ApiController
+      include LeadProviderAuthenticatable
+
+      def create
+        return head :no_content unless params[:id]
+
+        begin
+          user = User.find(params[:id])
+          early_career_teacher = user.early_career_teacher_profile
+          ParticipantAddService.call(early_career_teacher)
+          head :created
+        rescue ActiveRecord::RecordNotFound
+          head :unprocessable_entity
+        end
```

Comment:
I wonder if it needs to be in `begin rescue end` pattern. Maybe we can do it by using `def rescue end`?

Reply:
I think it should return 404 for RecordNotFound?

Reply:
Ah reading further, you probably were more referring to the 422 than the 204. The thought was that if it got that far, then the call was syntactically correct, but created an error. It was something that I'd used, and seen used for API calls with correct data structure, but bad data. I'm open to changing it.